src/platform/qt/AboutDialog.cpp

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2013-2015 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "AboutDialog.h"
+
+#include <QDesktopServices>
+#include <QUrl>
```

Comment:
Oh sorry, try doing that to `m_ui.logo->pixmap()->setDevicePixelRatio()`—it's a property of the pixmap.